lib/manageiq/session.rb

```diff
@@ -68,7 +68,7 @@ def self.revoke(*session_ids)
     # Create a new one each time instead of memoizing to avoid data getting
     # attached to the request hash (env).
     def self.fake_request
-      FakeRequest.new({})
+      FakeRequest.new({'rack.multithread' => true})
```

Comment:
I assume we want this for every situation this is used, right?  Can't think of a use case where having this would hurt, so I don't see it being a problem.

Reply:
yeah, exactly... since we use a threaded web server by default, I thought it was a good default.